bundles/ch.elexis.core/src/ch/elexis/core/services/IStockService.java

```diff
@@ -131,14 +132,16 @@ public enum Availability {
 
 	/**
 	 * Perform a single disposal of an article. The article will be withdrawn from
-	 * the Stock with the highest priority owning this article (if multiple).
+	 * the Stock with the highest priority owning this article (if multiple) or from
+	 * the given stock.
```

Comment:
* Was wenn stock provided und in diesem Stock nicht verfügbar? IStatus.WARN?
* Original-Methoden behalten und neue erzeugen - ansonsten bricht evtl. woanders der Code.